playground/src/pages/InstructionTable.tsx

```diff
@@ -0,0 +1,109 @@
+import { Fragment } from 'react';
+import ReactMarkdown from 'react-markdown';
+import gfm from 'remark-gfm';
+import math from 'remark-math';
+import katex from 'rehype-katex'
+import { assemblerInstructions } from '../data/instructions';
+
+import 'katex/dist/katex.min.css'
+
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(' ')
+}
+
+/** TODO: 
+ * Search button does not work yet 
+ * Expand button does not work yet - either remove or implement
+ * Decide on a table layout and pagination
+ * Table loads slowly - consider loading only the first 10 rows and then loading more as the user scrolls 
+ * or use react table (https://react-table.tanstack.com/docs/overview)`
+*/
+export default function InstructionTable() {
+  return (
+    <div className="px-4 sm:px-6 lg:px-8">
+      <div className="sm:flex sm:items-center">
+        <div className="sm:flex-auto">
+          <h1 className="text-xl font-semibold text-gray-900">Instructions</h1>
+          <p className="mt-2 text-sm text-gray-700">
+            A list of all Miden assembly instructions.
+          </p>
+        </div>
+        <div className="mt-4 sm:mt-0 sm:ml-16 sm:flex-none">
+        
+          <button
+            type="button"
+            className="inline-flex items-center justify-center rounded-md border border-transparent bg-indigo-600 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2 sm:w-auto"
+          >
+            Search
+          </button>
+        </div>
+      </div>
+      <div className="mt-8 flex flex-col">
```

Comment:
I'm not really familiar with the CSS, but on my machine this is rendering with a horizontal scrollbar. It would be nice to avoid it.

Reply:
jup, is removed

Reply:
The horizontal scrolling is still an issue. Maybe it was another div that is adding it?

Reply:
can you reload from your browser? It is removed here locally

Reply:
The horizontal bar is gone. But the `Expand` button is not working, it is just an anchor to the top of the page. Will that be done on a follow up?

Reply:
I would remove these buttons that are not working properly (the expand and the search button). Otherwise things feel broken.

Reply:
Good point. I have removed it.